src/katgpucbf/fgpu/ddc.py

```diff
@@ -60,8 +55,6 @@ def __init__(
             raise ValueError("taps must be positive")
         if subsampling <= 0:
             raise ValueError("subsampling must be positive")
-        if taps % subsampling != 0:
-            raise ValueError("taps must be a multiple of subsampling")
```

Comment:
I should probably read through the documentation update properly, but if you have a quick soundbite to share
* Any particular reason that this requirement is no longer necessary?

Reply:
I realised it would be quite easy to make the new kernel work without that restriction.